cfgov/apache/conf.d/alias.conf

```diff
@@ -10,7 +10,5 @@ Alias /lln8595c61g9qnvuwvtlcwo1k6kem8.html ${STATIC_PATH}/lln8595c61g9qnvuwvtlcw
 # Akamai Sureroute
 Alias /akamai/sureroute-test-object.html ${STATIC_PATH}/akamai-sureroute-test-object.html
 Alias /utilities/pages/akamai-sureroute-test-object.htm ${STATIC_PATH}/akamai-sureroute-test-object.html
-# Main static alias
-Alias /static/ ${STATIC_PATH}/
```

Comment:
Note above and below this line, Apache is still `Alias`ing these static files into the `STATIC_PATH` and in httpd.conf still has `STATIC_PATH` as its `DocumentRoot`. 

I'm leaving these alone for now because (a) Apache requires a `DocumentRoot` to run, and that's as good a one as any we could provide, and (b) rewriting these instead of file aliasing would seem to be part of the same problem of dealing with our redirects and rewrites, and could be tackled at the same time as migrating off of Apache.